README.md

```diff
@@ -22,148 +19,183 @@ Table of contents
-
-*Maven options:*
-* `-Dcucumber.options="--tags @smoketest"` - You can replace the tag with your own tag.
-* `-Dfailsafe.fork.count="1"` - The number of parallel tests run at the same time. The default is 1 and can be changed to other values depending on number of TestsIT classes.
-* `-Premote -Dremote.web.driver.url="<REMOTE DRIVER IP>"` - The `remote` parameter triggers tests to be executed using a remote VM. The `remote.web.driver.url` specifies the URL of the remote VM, for example `http://<IP_ADDRESS>:4444/wd/hub`.
-    * **Note:** The selenium grid must be setup so you can use this feature. Refer to the official documentation on the Selenium Grid.
-
-**Running subset of tests:**
-* Run a subset of tests in IntelliJ. Using the right mouse button (right-click), run any one of the TestsIT classes under `/selenium/src/test/java/com/elasticpath/cucumber/`.
-* Create your own local runner class to run your own tagged tests. For example, RunLocalTestsIT.java runs your own tagged tests @local.
-    * Do not commit the local runner class and tags. These are only for your local testing purposes.
-
-*Updating browser driver versions*
-* Download the latest browser driver from web. For example, chromedriver.
-* Update the RepositoryMap.xml for the driver version.
-* Hash value can be found locally if you run following in bash command locally.
-```
-openssl sha1 <filename>
-```
-* Example: https://github.com/Ardesco/Selenium-Maven-Template/blob/master/src/test/resources/RepositoryMap.xml
+### Adding or Editing Data
+
+To add or edit *mock data*:
+
+1. Start *Online* and perform the *flow* that you want to mock.<br/>
+2. In the browser, to see requests made by the *flow*, go to the **Settings>Tools>Developer Tools>Network** tab.
+3. Filter the requests using the **XHRF** or **XHR and Fetch** options.
+4. Copy the response directly from the request into a *.json* file at the            
+  `./src/offlineData` directory.  
+4. In the `.src/utils/Mock.js` file, add the following variable for data:
+  `const myData = require('../offlineData/myData.json')`
+5. Add data into the `mockData.set(myData.self.uri, { status: myStatusCode, data: myData}` map.
+    * For **followlocation**, create a new variable for the request uri and use that variable instead of the `myData.self.uri` variable. The responses include the *followed* url instead of the *request* url.
+    * For a request with no response, add the request URL with a status code and empty data. For more information, see the forms in the *Mock.js* file.
+
+### Verifying Data
+
+After mocking data for *flow*, enable [Offline Mode](#configuration) mode, restart the server and verify that the storefront is functioning as required.<br/>
+Go through your *flow* and verify everything works the same as Online. For any missing response, an error is displayed in the *browser console*, which includes the request for which mock data is not found. Mock that request to continue.
+
+### Default flows
```

Comment:
The entire section "Default flows" is outdated as we're not going to ship sample mock data. This section can be removed.